kola/tests/systemd/sysext.go

```diff
@@ -84,7 +84,7 @@ func checkSysextCustomDocker(c cluster.TestCluster) {
 	// First assert that Docker doesn't work because Torcx is disabled
 	_ = c.MustSSH(c.Machines()[0], cmdNotWorking)
 	// We build a custom sysext image locally because we don't host them somewhere yet
-	_ = c.MustSSH(c.Machines()[0], `git clone https://github.com/flatcar/sysext-bakery.git && git -C sysext-bakery checkout 2d6ab6bdba96476138c7db841e9a4c4ed38bd3fc`)
+	_ = c.MustSSH(c.Machines()[0], `git clone https://github.com/flatcar/sysext-bakery.git && git -C sysext-bakery checkout e68d2fe25c8412f4774477d1d75c40f615145c46`)
```

Comment:
Why do we even bother with checking out a specific commit?

Reply:
The commit ID was for consistent kola test behavior, I'm fine with git tags/releases as well :)

Reply:
For predictable behavior one could also take the last commit since the Flatcar build date but that relation is maybe a bit confusing.